### DISCUSSION TITLE
posts/redirect-404-pages-in-javascript/

### DISCUSSION THREAD

**Discussion by giscus** [dcangulo/davidangulo.xyz#156] [Announcements]
# posts/redirect-404-pages-in-javascript/

I recently migrated this blog from WordPress to Jekyll but due to the differences in permalinks, some posts now appear as 404. My old permalink for post is /:slug while it’s now /posts/:slug.

https://www.davidangulo.xyz/posts/redirect-404-pages-in-javascript/

**Comment by crystalsilvas** [comment]
Javascript is not my strong suit, but I have a client that recently moved to a new all in one CRM/Funnel/Website platform that doesn't support redirects. Would this sort of script work to parse our the url slug on the 404 pages to redirect to a subdomain where the blogs are? (basically the CRM only supports domain connection via nameservers and we were keeping the posts in wordpress, so we created a subdomain for the wordpress blog and are using the primary domain for website & funnels, but have pinterest pins with URLs that cannot be changed, resulting in 404 pages)
